src/sally/core/serving.py

```diff
@@ -52,7 +53,12 @@ def setup(hby, *, alias, httpPort, hook, auth, listen=False, timeout=10, retry=3
     reger = viring.Reger(name=hab.name, db=hab.db, temp=False)
     rep = storing.Respondant(hby=hby, mbx=mbx)
     verifier = verifying.Verifier(hby=hby, reger=reger)
+
     cdb = basing.CueBaser(name=hby.name)
+    if env_var_to_bool("CLEAR_ESCROWS", False):
```

Comment:
I don't think this needs to be configurable - but at least make the default true - escrows should be ephemeral

Reply:
They should be cleared from escrow with timeouts anyway - those timeouts should be short

Reply:
This now defaults to true.